editions/tw5.com/tiddlers/$__editions_tw5.com_more-info-view-template.tid

```diff
@@ -0,0 +1,19 @@
+created: 20250326000304980
+list-after: $:/core/ui/ViewTemplate/body
+modified: 20250327020654861
+tags: $:/tags/ViewTemplate
+title: $:/editions/tw5.com/more-info-view-template
+type: text/vnd.tiddlywiki
+
+<% if [<currentTiddler>!is[system]!skip-footer[yes]] %>
+
+<div class="tc-more-info-footer">
```

Comment:
IMO it should be `class="tc-tiddler-footer"`

Reply:
I have no problem with that, so long as we want to claim the term for this.